docs/DeveloperGuide.md

```diff
@@ -527,76 +525,200 @@ Use case ends
+   Use case resumes at step 2.
 
 **Use case: UC04 Help**
 
 **MSS**
 1. User requests help.
-2. McGymmy shows all commands and examples of command usages.
-
+2. McGymmy shows all commands, and the usage of each command.<br>
 Use case ends.
 
 **Extensions**
 No extensions.
 
-**Use case: UC05 Update food**
+**Use case: UC05 Edit food**
 
 **MSS**
-1. User requests to list food. (UC01)
+1. User requests to list food (UC01).
 2. McGymmy shows a list of food.
-3. User request to update a specific food on the list.
+3. User request to edit a specific food on the list.
 4. McGymmy updates the food.
 
 Use case ends.
 
 **Extensions**
-- 2a. The list is empty.<br>
-    Use case ends.
+- 2a. The list is empty<br>
+Use case ends.
+
 - 3a. The given index is invalid.<br>
-   - 3a1. McGymmy shows an error message.
-    Use case resumes at step 2.
+   - 3a1. McGymmy shows an error message.<br>
+   Use case resumes at step 2.
 
-**Use case: UC06 Add a macro command**
+- 3b. The format of edit method is invalid.
+   - 3b1. McGymmy shows an error message.<br>
+   Use case ends.
+    
+**Use case: UC06 Tag food**
 
 **MSS**
+1. User requests to list food (UC01).
+2. McGymmy shows a list of food.
+3. User request to tag a specific food on the list.
+4. McGymmy updates the food with the given tag.<br>
+Use case ends.
 
-1. User creates a macro to execute two 'add' commands in sequence.
-2. McGymmy adds the macro to the list of available commands.
-3. User uses the newly added macro command.
-4. McGymmy executes the two commands consecutively.
+**Extensions**
+- 2a. The list is empty.<br>
+   Use case ends.
+    
+- 3a. The given index is invalid.<br>
+   - 3a1. McGymmy shows an error message.<br>
+   Use case resumes at step 2.
 
+- 3b. The given tag already exists for the selected food.<br>
+   - 3a1. McGymmy shows an error message.<br>
+   Use case resumes at step 2.
+
+- 3c. The format of tag method is invalid.
+   - 3c1. McGymmy shows an error message.<br>
+   Use case ends.
+    
+**Use case: UC07 Untag food**
+
+**MSS**
+1. User requests to list food (UC01).
+2. McGymmy shows a list of food.
+3. User request to remove tag from a specific food on the list.
+4. McGymmy removes the tag from the food.<br>
 Use case ends
 
 **Extensions**
+- 2a. The list is empty.<br>
+   Use case ends.
 
- - 1a. The format of the macro is invalid.
-    - 1a1. McGymmy shows an error message.
+- 3a. The given index is invalid.<br>
+   - 3a1. McGymmy shows an error message.<br>
+   Use case resumes at step 2.
+
+- 3b. The given tag does not exist for the selected food.<br>
+   - 3a1. McGymmy shows an error message.<br>
+   Use case resumes at step 2.
+
+- 3c. The format of untag method is invalid.
+   - 3c1. McGymmy shows an error message.<br>
+   Use case ends.
+    
+**Use case: UC08 Find**
+
+**MSS**
+1. User requests to find (filter) food items based on certain criteria.
+2. McGymmy shows an updated list based on the filter criteria.
+
+Use case ends.
```

Comment:
Should have no break line before this line

Reply:
Same think here